### PR TITLE
US-3.13.5: Display comments in task drawer

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

**Comment by azayati** [comment]
Related [task PR](https://github.com/exo-addons/task/pull/184)

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <v-list-item-avatar size="30" class="mt-0">
+      <v-img :src="getUserAvatar(comment.author.username)"/>
+    </v-list-item-avatar>
+    <v-list-item-content 
+      class="pt-1 pb-0">
+      <v-flex xs12>
+        <v-layout>
+          <v-flex xs5>
+            <v-list-item-title
+              style="color: #578DC9;font-weight: bold;font-size: 14px"
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue]: Why not a class here ?
Should remove all embedded styles

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+              color="#578DC9"
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue]: The color attribute in css class is not sufficient ?

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            },
+        },
+        data() {
+          return {
+            hover : false,
+          }
+        },
+        computed: {
+            relativeTime() {
+                return this.getRelativeTime(this.comment.createdTime.time)
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue]:
```suggestion
                return this.getRelativeTime(this.comment.createdTime.time);
```

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    </v-list-item-avatar>
+    <v-list-item-content
+      class="pt-1 pb-0"
+      @mouseover="hover = true"
+      @mouseleave="hover = false">
+      <v-flex xs12>
+        <v-layout>
+          <v-flex xs5>
+            <v-list-item-title
+              style="color: #578DC9;font-weight: bold;font-size: 14px"
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue]: same above remark

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue [hunk 5]
```diff
@@ -0,0 +1,99 @@
+<template>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue]: Can this component be unified with the TaksComments component ?with a parameter ?

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue (hunk 1 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue]: style removed

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue (hunk 2 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue]: no there is not attribute color for v-list-item-title,It was my fault

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue (hunk 3 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskComments.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue (hunk 4 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue (hunk 5 above)
Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue]: IMO ,no ,since they don't have quite the same structure

Comment (azayati) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskSubComments.vue]: ok to be done on next PR

**Pull request merged**